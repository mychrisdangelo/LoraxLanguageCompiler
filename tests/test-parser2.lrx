int do() {
print(1);
}

int do2() {
print(2);
}

int main()
{
do();
do2();
}
