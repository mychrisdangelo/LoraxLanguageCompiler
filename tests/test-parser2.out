int main()
{
do();
do2();
}

int do2()
{
print(2);
}

int do()
{
print(1);
}
